sql/core/src/test/scala/org/apache/spark/sql/execution/ui/SQLListenerSuite.scala

```diff
@@ -147,6 +147,10 @@ class SQLListenerSuite extends SparkFunSuite with SharedSQLContext with JsonTest
 
     checkAnswer(listener.getExecutionMetrics(0), accumulatorUpdates.mapValues(_ * 2))
 
+    // Non-existing driver accumulator updates should be filtered and no exception will be thrown.
```

Comment:
I was doing some own experiments that adds physical operators at runtime. The metrics of the added operators are not registered to the execution so I got a NoSuchElementException.